package.json

```diff
@@ -39,7 +39,9 @@
   },
   "dependencies": {
     "@turf/bbox": "^6.0.1",
+    "cached-path-relative": "^1.0.2",
     "lodash": "^4.17.10",
+    "randomatic": "^3.1.1",
```

Comment:
are these direct dependencies or pulled in from an upstream library?

Reply:
Looks like they are pulled in from an upstream library @andrewharvey is there a better way to update them?

Reply:
I think (but I'm not expert) `npm audit fix --force` is the proper way to do it (apart from getting the upstream libraries to fix their dependencies).

This way it only changes the `package-lock.json`.

Reply:
@andrewharvey thanks! That makes sense. I reverted my changes and tried `npm audit fix --force` but nothing changed in my `package-lock.json`. Any ideas?

Reply:
I think update just budo should do the trick @andrewharvey 👍

Reply:
:+1: Yes that did it for me, in combination with `npm audit fix`. Then `npm audit` says there are no more issues. You can drop randomatic and cached-path-relative from the direct dependencies now.